test/test_uri_parser.py

```diff
@@ -24,6 +24,7 @@
 sys.path[0:0] = [""]
 
 from test import unittest
+from unittest.mock import MagicMock, patch
```

Comment:
I would have expected this test to be async. Can you check if we already have a ticket for converting this file and open one if we don't?

Reply:
IIRC, @NoahStapp said we don't need an async test …

Reply:
How's that? Don't we need to test the async code path for SRV parsing?

Reply:
🤷  I'll follow up on Slack and open a ticket as needed